mllib/src/main/scala/org/apache/spark/ml/ann/Layer.scala

```diff
@@ -774,8 +640,8 @@ private[ml] class FeedForwardTrainer(
     val inputSize: Int,
     val outputSize: Int) extends Serializable {
 
-  // TODO: what if we need to pass random seed?
-  private var _weights = topology.getInstance(11L).weights()
+  private var _seed = 11L
```

Comment:
We use `11L` in many places. It would be nice to generate a default seed based on the class name, e.g., `this.getClass.getName.hashCode.toLong`.